config/rbac/role.yaml

```diff
@@ -32,6 +32,32 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - cluster.open-cluster-management.io
+  resources:
+  - backupschedules
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - cluster.open-cluster-management.io
+  resources:
+  - backupschedules/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - cluster.open-cluster-management.io
+  resources:
+  - backupschedules/status
+  verbs:
+  - get
+  - patch
+  - update
 - apiGroups:
```

Comment:
We don't need to `get,list,watch`? 

Here we're missing the tags I've just added in the fix for RBAC... Same.
Add `kubebuilder` tags for rabc for `managedclusters.cluster.open-cluster-management.io`